console/lockable_trait.rst

```diff
@@ -30,7 +30,7 @@ that adds two convenient methods to lock and release commands::
             }
 
             // If you prefer to wait until the lock is released, use this:
-            // $this->lock(true);
+            // $this->lock('lockname', true);
```

Comment:
What about passing  `null` as the first argument instead to use the default lock name?